Cargo.toml

```
@@ -12,7 +12,7 @@ default = ["osmosis"]
 osmosis = []
 
 [dependencies]
-osmosis-std = { git = "https://github.com/osmosis-labs/osmosis-rust", rev = "3ea06eda3e5f2979a6232a118544d292a3bfca9f" }
+osmosis-std = { git = "https://github.com/osmosis-labs/osmosis-rust", branch = "osmosis-main-clone" }
```

Let's set this to latest commit on master
```suggestion
osmosis-std = { rev = "b3df85158eb52ad5d99f3580f4a128bbdaeffa17", git = "https://github.com/osmosis-labs/osmosis-rust.git" }
```